client/state/data-layer/wpcom/me/index.js

```diff
@@ -25,6 +26,7 @@ registerHandlers(
 	mergeHandlers(
 		account,
 		block,
+		blocks,
```

Comment:
Do we want to name this one `siteBlocks`? I'd be confused what `blocks` is in the context of `/me`.

Reply:
There are some exceptions, especially if we're skipping a level as I suggested in another comment. You can see several examples:

* https://github.com/Automattic/wp-calypso/blob/d84b6f75acc836c6727be8ba19771528c3a77269/client/state/data-layer/wpcom/index.js#L18
* https://github.com/Automattic/wp-calypso/blob/d84b6f75acc836c6727be8ba19771528c3a77269/client/state/data-layer/wpcom/index.js#L20
* https://github.com/Automattic/wp-calypso/blob/d84b6f75acc836c6727be8ba19771528c3a77269/client/state/data-layer/wpcom/index.js#L31

Following these, it would probably make sense to name it `sitesBlocks`.

Reply:
That's fair enough. I've renamed to `siteBlocks` 👍